app/controllers/user_research_recruitment_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UserResearchRecruitmentController < ApplicationController
```

Comment:
This commit's the first one where I get the feeling that I'm following the development process and not seeing the feature fall into place step-by-step.

Reply:
I've resolved this by curating the commits in #2301.